src/main/java/org/semux/config/AbstractConfig.java

```diff
@@ -120,6 +120,9 @@
     protected boolean vmEnabled = false;
     protected int vmMaxStackSize = 1024;
     protected int vmInitHeapSize = 128;
+    protected int vmBlockGasLimit = 999_999;
+    protected int vmMaxBlockGasLimit = 9_999_999;
+    protected int vmMinGasPrice;
```

Comment:
Missing default value for min_gas_price?

Question: are we going to have a dynamic or fixed block gas limit, from the protocol perspective? I don't think this is something a client should be able modify.

I feel like `min_gas_price` is a pending manage (txpool) configuration, unless we want to enforce this on a protocol level.

Reply:
Since we don't have mining or concept of concensus on gas limit, it feels like a validators limit as to how much gas it wants to process and how much it wants to charge for gas.  If I have 16 cores and 64 gb of RAM I may want to have a low limit and process everything, but if I have a trash tier cheap host, I may want to charge more and have Lower limit.  But mostly this is to allow us to tune and find reasonable limits in vm without causing testnet to get stuck if we push perf test too far.  Allowing control of these means testnet can continue to forge. And we can tune what gas price makes sense

Reply:
We do still need to add sorting of pending txes by fee and take nonce into consideration.  Can be post testnet as it's not concensus breaking xhange